### PR TITLE
Updates/moves sitemap_spec

### PR DESCRIPTION
**Pull request by filipefurtad0**
#### What? Why?

Continues #7929.

Moves/updates the last :tada: consumer (feature) spec into system spec, sitemap_spec.rb.

#### What should we test?

Green build.

#### Release notes

Changelog Category: Technical changes

Moves/updates sitemap_spec.rb into system.

### CODE REVIEW

File: spec/system/consumer/sitemap_spec.rb [hunk 1]
```diff
@@ -8,7 +8,16 @@
...
   end
 end
+
+private
+
+def return_page(website)
+  # routing does not include the port of the session, this method adds it
+  url = URI(page.driver.browser.url)
+  path = URI(website).path
+  return_page = "http://" + url.host.to_s + ':' + url.port.to_s + path
```
Comment (filipefurtad0) [inline spec/system/consumer/sitemap_spec.rb:22]: On this line, Rubocop seems to prefer:
`  return_page = "http://" + "#{url.host}" ":" + url.port.to_s + path`

But tbh, I don't think it makes much sense to mix the methods of converting integers to strings...

Comment (mkllnk) [reply spec/system/consumer/sitemap_spec.rb:22]: No, it wants this:

```suggestion
  return_page = "http://#{url.host}:#{url.port}#{path}"
```

**Review by mkllnk — COMMENTED** [review]
Great! Just the style left.

**Comment by filipefurtad0** [comment]
Thanks Maikel :pray: 
Fixed the rubocop issue, build is green - moving to code review :+1:

**Review by jibees — APPROVED** [review]
Should we merge it directly?

**Review by mkllnk — APPROVED** [review]

**Pull request merged**